docs/release-notes/2773-work-dir.txt

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**New Features**
+
+-  Support configuring working directory in the ``environment.work_dir`` configuration for all task
```

Comment:
@shiyuann is this PR blocked on another (since title only describes the other feature) or am I to review both? either is fine just curious

Reply:
It's not blocked by anyone. It's to configure TensorBoard idle timeout, which we already support. But this will also help if we support Notebook idle timeout.

Reply:
But yes, it will be rebased if #2773 is merged. You can review the last two commits.